test/integration/ember-i18n-csv-test.js

```diff
@@ -56,6 +64,23 @@ describe('integration - ember-i18n-csv', function() {
         });
       });
     });
+    describe('merge', function() {
+      beforeEach(function() {
+        fs.copySync('test/fixtures/locales', 'tmp/locales', { clobber: true });
+      });
+
+      afterEach(function() {
+        fs.emptyDirSync('tmp');
+      });
+
+      it('handles option --merge', function() {
+        return emberI18nCsv('to-js', 'tmp/locales', 'test/fixtures/i18n-merge.csv', { merge: true }).then(() => {
+          return areDirsEqual('tmp/locales', 'test/fixtures/locales').then(areSame => {
```

Comment:
I'm testing that the merge does not clobber the translations that are present already but not in the .csv file. This is successful because the .csv is a subset of the tmp/locales.  Should i add an additional test that checks to see if additions are properly added?